News/Entities/Models/CommentModel.cs

```diff
@@ -2,6 +2,8 @@
 
 public class CommentModel
 {
+    public Guid Id { get; set; }
+
     public string Comment { get; set; }
```

Comment:
Comment => content? or it's out of scope of this PR?

Reply:
Will be updated soon. I have a few questions about it.